### ISSUE TITLE
[docs] Strange order of items in docs contents in left pane

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#33055]
```
torch.optim
Quantization
Distributed RPC Framework
torch.random
```
why this order? not even alphabetical. I'd suggest that big special topics such as Quantization are presented on top in a separate section, and the rest to be in alphabetical order.

![image](https://user-images.githubusercontent.com/1041752/73972687-8c035b00-4921-11ea-82a9-01c4d09cc5b3.png)

**Comment by fmassa** [comment]
I agree, the current order in the documentation is weird and should be improved.

cc @brianjo

**Comment by brianjo** [comment]
@jlin27 Can you take a look at updating this per the discussion in this thread? Thanks!

**Comment by jlin27** [comment]
Will discuss this w/ Joe, Bruce, and the rest of the team at the next Weekly Docs Sync

**Comment by vadimkantorov** [comment]
Despite https://github.com/pytorch/pytorch/pull/34928, the items are still in mixed order
